src/views/course-edit-tools/course-preferences.vue

```diff
@@ -32,26 +32,6 @@
           :title="y18n('coursePreferences.enrollment.hint')"
         ></i>
       </div>
-      <div class="col text-dark">
```

Comment:
what if you later decided to not want enrollement? (or do?) - if you remove this, there is no option to change enrollment once you created a course